source/nodejs/adaptivecards-designer/src/designer-peers.ts

```diff
@@ -3234,8 +3267,13 @@ export class TablePeer extends TypedCardElementPeer<Adaptive.Table> {
 
                         this.cardElement.addRow(row);
 
-                        this.insertChild(CardDesignerSurface.cardElementPeerRegistry.createPeerInstance(this.designerSurface, this, row));
-                        this.updateChildren();
+                        const newPeer = CardDesignerSurface.cardElementPeerRegistry.createPeerInstance(this.designerSurface, this, row);
+                        newPeer.insertAfterNeighbor = true;
+
+                        this.insertChild(newPeer);
+
+                        // TODO: I'm not sure if this call is necessary
+                        // this.updateChildren();
```

Comment:
Does anyone know why we call `updateChildren()` here?

Reply:
@dclaux Do you have any info about this?

Reply:
I'm leaving this line commented out since I don't see a behavior difference in the designer and other similar elements (`ColumnSet`, `ImageSet`) do not call `updateChildren()`